File: BackEnd/Service/Models/SkillModel.cs
namespace Service.Models
{
    public class SkillModel
    {
        public Guid SkillId { get; set; }

        public string SkillName { get; set; } 

        public string? Description { get; set; }

        public bool IsDeleted { get; set; } = false;
    }
}

File: FrontEnd/src/app/data/skill/skill.model.ts
export class Skill {
	skillId?: string;
	skillName?: string;
	description?: string;
	isDeleted?: boolean;
}

export class SkillAddModel {
	skillName: string = "";
	description?: string;
}

export class SkillUpdateModel {
	skillName: string = "";
	description?: string;
}
